client.go

```diff
@@ -103,8 +105,17 @@ func (c *Client) execute(req *Request) (*Response, error) {
 	}
 
 	var resp []byte
+
+	// we don't support block hash and overrides at the same time
+	if req.BlockHash != zeroHash && req.Overrides != nil {
```

Comment:
```suggestion
	if req.BlockHash != zeroHash && len(req.Overrides) > 0 {
```

Reply:
I think check `nil` is correct because `req.Overrides` is a pointer.